coq/README.md

```diff
@@ -0,0 +1,15 @@
+## Dependencies
+
+The coq libraries uses `compcert` for machine signed and unsigned integer modulo arithmetic, and `coqprime` for finite field arithmetic on prime modulus (to support hacspec's `nat_mod p` type). These can be installed through `opam`:
+
+```
+opam install coq-compcert coq-coqprime
+```
+(assuming you have coq installed through opam).
+
+## Compiling the coq files
+
+In folder `/coq`, type `make`. This compiles the coq libraries and the compiled examples, as defined in `_CoqProject`.
+This requires the coq compiler to be installed (only tested on coq 8.13.1)
```

Comment:
Maybe you can fix the coq version in some way?

Reply:
Do you mean in the build somehow, or just here in the readme?

Reply:
I mean in the build. It would be nice if we could have an easy way for people to set up everything in a way that doesn't lead to errors due to things like the wrong coq version.